components/dashboard/src/projects/Prebuilds.tsx

```diff
@@ -190,7 +190,7 @@ export default function (props: { project?: Project; isAdminDashboard?: boolean
                 </div>
                 <ItemsList className="mt-2">
                     <Item header={true}>
-                        <ItemField className="my-auto w-5/12">
+                        <ItemField className="my-auto md:w-3/12 xl:w-4/12">
```

Comment:
It would be great to not overshoot at `md`.

Reply:
Tip for the future: Using CSS grid is much more powerful and robust in these situation than trying to guess good fixed widths.

With grid, you can simply specify "I want three columns of exactly the same width", or specify custom relative column widths, but they'll almost always be correct regardless of screen size, margins, size of content, etc.